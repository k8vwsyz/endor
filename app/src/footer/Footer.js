import React from "react"
import './footer.css'
import { Container, Grid, Typography } from '@material-ui/core'
import { makeStyles } from '@material-ui/core/styles'


const useStyles = makeStyles((theme) => ({
  footer: {
    padding: 8,
    background: theme.palette.primary.light
  },
  text: {
    color: theme.palette.primary.contrastText
  }
}))

const Footer = () => {
  const classes = useStyles()

  return (



    <footer className={classes.footer}>


      <Container
        maxWidth="lg">
        <Grid container spacing={2} >
          <Grid item xs={12} md={6} >
            <Typography variant="h5" className={classes.text} >NAVIGATE</Typography>
            <ul className="footer-ul">
              {/* eslint-disable-next-line */}
              <li><a href="https://www.gu.se/" target="_blank" className={classes.text}>Home</a></li>
              {/* eslint-disable-next-line */}
              <li><a href="https://twitter.com/goteborgsuni?ref_src=twsrc%5Egoogle%7Ctwcamp%5Eserp%7Ctwgr%5Eauthor" target="_blank" className={classes.text}>Twitter</a></li>
              {/* eslint-disable-next-line */}
              <li><a href="https://www.facebook.com/goteborgsuniversitet" target="_blank" className={classes.text}>Facebook</a></li>
            </ul>
          </Grid>
          <Grid item xs={12} md={4} >
            <Typography variant="h5" className={classes.text} >USEFUL LINKS</Typography>
            <ul className="footer-ul">
              {/* eslint-disable-next-line */}
              <li><a href="https://fn.se/globala-malen-for-hallbar-utveckling/" target="_blank" className={classes.text}>Sustainable Development Goals</a></li>
              {/* eslint-disable-next-line */}
              <li><a href="https://en.wikipedia.org/wiki/Endor_(Star_Wars)" target="_blank" className={classes.text}>Endor</a></li>
              {/* eslint-disable-next-line */}
              <li><a href="https://www.wateraid.org/facts-and-statistics" target="_blank" className={classes.text}>Wateraid - facts and statistics</a></li>
            </ul>
          </Grid>
        </Grid>
      </Container>
    </footer>
  )
}

export default Footer