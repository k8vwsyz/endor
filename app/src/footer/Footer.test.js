import React from "react";
import { render, cleanup } from '@testing-library/react'

import Footer from "./Footer";

afterEach(cleanup);

it('snapshot test of page', () => {

  const { asFragment } = render(
    <Footer />
  )

  expect(asFragment()).toMatchSnapshot()
});