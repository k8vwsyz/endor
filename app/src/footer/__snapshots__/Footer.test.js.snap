// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test of page 1`] = `
<DocumentFragment>
  <footer
    class="makeStyles-footer-1"
  >
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg"
    >
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2"
      >
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-6"
        >
          <h5
            class="MuiTypography-root makeStyles-text-2 MuiTypography-h5"
          >
            NAVIGATE
          </h5>
          <ul
            class="footer-ul"
          >
            <li>
              <a
                class="makeStyles-text-2"
                href="https://www.gu.se/"
                target="_blank"
              >
                Home
              </a>
            </li>
            <li>
              <a
                class="makeStyles-text-2"
                href="https://twitter.com/goteborgsuni?ref_src=twsrc%5Egoogle%7Ctwcamp%5Eserp%7Ctwgr%5Eauthor"
                target="_blank"
              >
                Twitter
              </a>
            </li>
            <li>
              <a
                class="makeStyles-text-2"
                href="https://www.facebook.com/goteborgsuniversitet"
                target="_blank"
              >
                Facebook
              </a>
            </li>
          </ul>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-4"
        >
          <h5
            class="MuiTypography-root makeStyles-text-2 MuiTypography-h5"
          >
            USEFUL LINKS
          </h5>
          <ul
            class="footer-ul"
          >
            <li>
              <a
                class="makeStyles-text-2"
                href="https://fn.se/globala-malen-for-hallbar-utveckling/"
                target="_blank"
              >
                Sustainable Development Goals
              </a>
            </li>
            <li>
              <a
                class="makeStyles-text-2"
                href="https://en.wikipedia.org/wiki/Endor_(Star_Wars)"
                target="_blank"
              >
                Endor
              </a>
            </li>
            <li>
              <a
                class="makeStyles-text-2"
                href="https://www.wateraid.org/facts-and-statistics"
                target="_blank"
              >
                Wateraid - facts and statistics
              </a>
            </li>
          </ul>
        </div>
      </div>
    </div>
  </footer>
</DocumentFragment>
`;
